client/js/widgets/button.js

```diff
@@ -275,14 +293,17 @@ class Button extends Widget {
       }
 
       if(this.p('debug')) {
-        $('#debugButtonOutput').textContent += '\n\n\nOPERATION: \n' + JSON.stringify(a, null, '  ');
+        var msg = ''
```

Comment:
Not really important but please use const or let instead of var. No need to change it though.

Reply:
These nuances with javascript I'm not so familiar with, so please be picky to help me understand subtle importances. :-) I changed it to `let`.

Reply:
Yeah, I only learned about those recently as well. `var` has a pretty weird scope. `let` and `const` are valid inside their block which is what I would expect.